rust/bridge/shared/src/jni/convert.rs

```diff
@@ -164,6 +164,14 @@ impl<'a> SimpleArgTypeInfo<'a> for Option<u32> {
     }
 }
 
+/// Reinterprets the bits of the jlong as a u64.
```

Comment:
Do you want it to be consistent with others like:

Reinterprets the bits of the Java \`long\` as a \`u64\`.